ProcessMaker/Filters/CasesFilter.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace ProcessMaker\Filters;
+
+class CasesFilter extends BaseFilter
+{
+    public const TYPE_STATUS = 'Status';
```

Comment:
Seems this const was already declared in the `BaseFilter` with the same value, is this line required?